internal/cli/cmd/clustermesh.go

```diff
@@ -336,3 +346,49 @@ func newCmdExternalWorkloadStatus() *cobra.Command {
 
 	return cmd
 }
+
+func newCmdClusterMeshEnableWithHelm() *cobra.Command {
+	var params = clustermesh.Parameters{
+		Writer: os.Stdout,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "enable",
+		Short: "Enable ClusterMesh ability in a cluster using Helm",
+		Long:  ``,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			params.Namespace = namespace
+			ctx := context.Background()
+			if err := clustermesh.EnableWithHelm(ctx, k8sClient, params); err != nil {
+				fatalf("Unable to enable ClusterMesh: %s", err)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&params.ServiceType, "service-type", "LoadBalancer", "Type of Kubernetes service to expose control plane { ClusterIP | LoadBalancer | NodePort }")
```

Comment:
This changes the default value. Did you have any reason you wanted to change the default?